observability/observability.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021  The GoHBase Authors.  All rights reserved.
+// This file is part of GoHBase.
+// Use of this source code is governed by the Apache License 2.0
+// that can be found in the COPYING file.
+
+package observability
```

Comment:
Could we move this under gohbase/internal/observability instead, to remove it from the public API of GoHBase?